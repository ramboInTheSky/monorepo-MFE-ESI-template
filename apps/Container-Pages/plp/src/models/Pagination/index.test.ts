import {Pagination} from "."

describe("Model - Pagination: ", () => {
    it("should match the Pagination", () => {
        expect(new Pagination()).toMatchSnapshot()
    })
})
