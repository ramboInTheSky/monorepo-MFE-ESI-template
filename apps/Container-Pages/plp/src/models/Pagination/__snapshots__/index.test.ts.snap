// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model - Pagination:  should match the Pagination 1`] = `
Pagination {
  "nextPageUrl": "",
  "numberOfItems": 0,
  "previousPageUrl": "",
  "start": 0,
  "totalResults": 0,
}
`;
